tools/test.py

```diff
@@ -164,6 +168,7 @@ def main():
             for key in ['interval', 'tmpdir', 'start', 'gpu_collect']:
                 eval_kwargs.pop(key, None)
             eval_kwargs.update(dict(metric=args.eval, **kwargs))
+            eval_kwargs['bbox_kwargs'] = {'classwise': True}
```

Comment:
This seems an undesirable hard code in the main file.

Reply:
Sorry, my bad. I forgott to create a branch for the pull request.